src/ProductConstructionService/ProductConstructionService.BarViz/Components/CodeflowContextMenu.razor

```diff
@@ -0,0 +1,210 @@
+            <ChildContent>
+                Forward flow
+                <span slot="start">
+                    <FluentIcon Value="@(new Icons.Regular.Size16.ArrowRight())" Color="@Color.Neutral" />
+                </span>
+            </ChildContent>
+            <MenuItems>
+                <FluentMenuItem OnClick="@(() => ShowDetails(Codeflow.ForwardflowSubscription))">
+                    Show subscription
+                    <span slot="start">
+                        <FluentIcon Value="@(new Icons.Regular.Size16.DocumentOnePage())" Color="@Color.Neutral" />
+                    </span>
+                </FluentMenuItem>
+
+                <FluentMenuItem OnClick="@(() => Clipboard.SetTextAsync(Codeflow.ForwardflowSubscription.Id.ToString()))">
+                    Copy subscription ID
+                    <span slot="start">
+                        <FluentIcon Value="@(new Icons.Regular.Size16.Copy())" Color="@Color.Neutral" />
+                    </span>
+                </FluentMenuItem>
+
+                @if (Codeflow.ForwardflowSubscription.Enabled)
+                {
+                    <FluentMenuItem OnClick="@TriggerForwardflow">
+                        Trigger subscription
+                        <span slot="start">
+                            <FluentIcon Value="@(new Icons.Regular.Size16.Flash())" Color="@Color.Neutral" />
+                        </span>
+                    </FluentMenuItem>
+                }
+
+                @if (Codeflow.ForwardflowSubscription.LastAppliedBuild != null)
+                {
+                    <FluentMenuItem OnClick="@(async () => await JSRuntime.OpenNewWindow(ForwardflowBuildLink!))">
+                        Show last flown repo build
+                        <span slot="start">
+                            <FluentIcon Value="@(new Icons.Regular.Size16.Flashlight())" Color="@Color.Neutral" />
+                        </span>
+                    </FluentMenuItem>
+
+                    <FluentMenuItem OnClick="@(async () => await JSRuntime.OpenNewWindow(Codeflow.ForwardflowSubscription.LastAppliedBuild.GetBuildLink()))">
+                        Go to build
+                        <span slot="start">
+                            <FluentIcon Value="@(new Icons.Regular.Size16.CloudLink())" Color="@Color.Neutral" />
+                        </span>
+                    </FluentMenuItem>
+
+                    <FluentMenuItem OnClick="@(async () => await JSRuntime.OpenNewWindow(Codeflow.ForwardflowSubscription.LastAppliedBuild.GetCommitLink()))">
+                        Go to commit
+                        <span slot="start">
+                            <FluentIcon Value="@(new Icons.Regular.Size16.DocumentLink())" Color="@Color.Neutral" />
+                        </span>
+                    </FluentMenuItem>
+                }
+            </MenuItems>
+        </FluentMenuItem>
+
+        @if (Codeflow.ForwardflowSubscription.LastAppliedBuild != null)
```

Comment:
```suggestion
        @if (Codeflow.ForwardflowSubscription?.LastAppliedBuild != null)
```

Or if this is not needed then the nullability check of `ForwardflowSubscription` on line `145` should also not be needed, right?